packages/docs/src/components/DetailsBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export interface DetailsBlockProps {
+  summary: string;
+  children?: React.ReactNode;
```

Comment:
request to add an optional "id" prop here, to allow us to send links to the details blocks. 

This is really helpful for sharing examples with customers.

```tsx
<summary id={pops.id}>
  <a href={props.id}>{props.summary}</a>
</summary>
```

Reply:
I actually think I need to back out all of the `<details>` and `<summary>` changes, because they don't render correctly.

I don't quite understand why the `npx docusaurus-mdx-checker` tool complains about them.  1, they're valid JSX, and 2, they're a documented pattern, and 3, they don't work when used in a React component?

Reply:
That's fine. I was actually surprised you needed to this chagne in the first place

Reply:
It turns out it doesn't like it when `<details>` and `<summary>` are on the same line 🤷  Moving them to new lines fixed the warnings.  It must be something about those particular tags.